queries.sql

```diff
@@ -1,8 +1,33 @@
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT name FROM animals WHERE date_of_birth BETWEEN '2016/01/01' AND '2019/12/31';
-SELECT name FROM animals WHERE neutered=TRUE AND escape_attempts < 3;
-SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
-SELECT date_of_birth, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = TRUE;
-SELECT * FROM animals WHERE NOT name = 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT SPA1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SPA1;
+COMMIT;
+SELECT * FROM animals;
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230142173-bf648580-5f9d-4e49-916e-cfdffec41a90.png)

**[OPTIONAL]**

- [ ] Kindly make your Setup something like the below to make your readme file useful and descriptive 😊

### Setup
Clone this repository to your desired folder:
```
cd your-folder
git clone https://github.com/hakichris/SQLvetclinic.git
cd SQLvetclinic
```